sdks/java/core/src/main/java/org/apache/beam/sdk/schemas/transforms/Unnest.java

```diff
@@ -1,171 +0,0 @@
-/*
- * Licensed to the Apache Software Foundation (ASF) under one
- * or more contributor license agreements.  See the NOTICE file
- * distributed with this work for additional information
- * regarding copyright ownership.  The ASF licenses this file
- * to you under the Apache License, Version 2.0 (the
- * "License"); you may not use this file except in compliance
- * with the License.  You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package org.apache.beam.sdk.schemas.transforms;
-
-import com.google.auto.value.AutoValue;
-import java.util.List;
-import org.apache.beam.sdk.annotations.Experimental;
-import org.apache.beam.sdk.annotations.Experimental.Kind;
-import org.apache.beam.sdk.schemas.Schema;
-import org.apache.beam.sdk.schemas.Schema.Field;
-import org.apache.beam.sdk.transforms.DoFn;
-import org.apache.beam.sdk.transforms.PTransform;
-import org.apache.beam.sdk.transforms.ParDo;
-import org.apache.beam.sdk.transforms.SerializableFunction;
-import org.apache.beam.sdk.values.PCollection;
-import org.apache.beam.sdk.values.Row;
-import org.apache.beam.vendor.guava.v26_0_jre.com.google.common.collect.Lists;
-
-/**
- * A {@link PTransform} to unnest nested rows.
- *
- * <p>For example, consider a Row with the following nestedschema:
- *
- * <p>UserEvent Schema: userid: INT64 timestamp: DATETIME location: LatLong
- *
- * <p>LatLong Schema: latitude: DOUBLE longitude: DOUBLE
- *
- * <p>After unnesting, all of the rows will be converted to rows satisfying the following schema:
- *
- * <p>UserEvent Schema: userid: INT64 timestamp: DATETIME location.latitude: DOUBLE
- * location.longitude: DOUBLE
- *
- * <p>By default nested names are concatenated to generated the unnested name, however {@link
- * Unnest.Inner#withFieldNameFunction} can be used to specify a custom naming policy.
- *
- * <p>Note that currently array and map values are not unnested.
- */
-@Experimental(Kind.SCHEMAS)
-public class Unnest {
```

Comment:
I assume there's no concern with removing Unnest outright (rather than deprecating for a period and pointing to `Select.flattenedSchema`) since this is all Experimental?

Reply:
yeah - should be fine I think.